model/monitorCache/implementation/MonitoringStorage.php

```diff
@@ -803,13 +805,16 @@ public function getStatusesStatistic($limit = 10, $offset = 0, $orderby = 'label
 
         $queryBuilder->from(self::TABLE_NAME, 'delivery_m');
 
+        $paramsValues = [];
         foreach ($statusesMap as $label => $statusUri) {
             $queryBuilder->leftJoin(
                 'delivery_m',
                 self::TABLE_NAME,
                 $conn->quoteIdentifier('s_'.$label),
-                'delivery_m.delivery_execution_id='.$conn->quoteIdentifier('s_'.$label).'.delivery_execution_id and '.$conn->quoteIdentifier('s_'.$label).'.status = \''.$statusUri.'\''
+                'delivery_m.delivery_execution_id='.$conn->quoteIdentifier('s_'.$label).'.delivery_execution_id and '
+                .$conn->quoteIdentifier('s_'.$label).'.status = :status_uri'
             );
+            $paramsValues[':status_uri'] = $statusUri;
```

Comment:
you override ':status_uri' parameter on each iteration

Reply:
Good catch, thank you for the fix.